lntest/harness.go

```diff
@@ -171,6 +171,104 @@ func (h *HarnessTest) Context() context.Context {
 	return h.runCtx
 }
 
+// setupWatchOnlyNode initializes a node with the watch-only accounts of an
+// associated remote signing instance.
+func (h *HarnessTest) setupWatchOnlyNode(name string,
+	signerNode *node.HarnessNode, password []byte) *node.HarnessNode {
+
+	// Prepare arguments for watch-only node connected to the remote signer.
+	remoteSignerArgs := []string{
+		"--remotesigner.enable",
+		fmt.Sprintf("--remotesigner.rpchost=localhost:%d",
+			signerNode.Cfg.RPCPort),
+		fmt.Sprintf("--remotesigner.tlscertpath=%s",
+			signerNode.Cfg.TLSCertPath),
+		fmt.Sprintf("--remotesigner.macaroonpath=%s",
+			signerNode.Cfg.AdminMacPath),
+	}
+
+	// Fetch watch-only accounts from the signer node.
+	resp := signerNode.RPC.ListAccounts(&walletrpc.ListAccountsRequest{})
+	watchOnlyAccounts, err := walletrpc.AccountsToWatchOnly(resp.Accounts)
+	if err != nil {
+		h.Fatalf("unable to find watch only accounts for %s: %v",
+			name, err)
+	}
+
+	// Create a new watch-only node with remote signer configuration.
+	return h.NewNodeRemoteSigner(name, remoteSignerArgs, password,
```

Comment:
nit: formatting of multi-line function call.

Reply:
Updated ✅